### DISCUSSION TITLE
Github repo release notes in Mkdocs site

### DISCUSSION THREAD

**Discussion by GBBBAS** [squidfunk/mkdocs-material#5099] [Q&A] [answered]
Is there any plugin or option to pull in the release notes from a github repo and display it in a changelog on a mkdocs site?

I assume that for this repo, the [changelog.md](https://squidfunk.github.io/mkdocs-material/changelog/) file is manually populated? Could this be generated from the release notes in the github repo release section?

**Comment by squidfunk** [comment, marked as answer]
Yes, we populate the changelog manually. It certainly could be generated, e.g. by implementing a [hook](https://www.mkdocs.org/user-guide/configuration/#hooks) or plugin.
